.gitignore

```diff
@@ -4,6 +4,7 @@ node_modules
 .env
 .vercel
 .output
+.vscode/settings.json
```

Comment:
We want to allow every dev to define their own VSCode workspace settings. Hence, we avoid adding it to git.